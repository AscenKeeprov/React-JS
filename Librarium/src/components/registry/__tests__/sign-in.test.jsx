import Form from '../../../components/shared/form';
import React from 'react';
import { shallow } from 'enzyme';
import SignIn from '../sign-in';

describe('Sign in view', () => {
	let signInView = null;

	afterEach(() => {
		signInView = null;
	});

	beforeEach(() => {
		signInView = shallow(<SignIn />);
	});

	it('renders without crashing', () => {
		expect(signInView).toMatchSnapshot();
	});

	it('contains one form component', () => {
		expect(signInView.html()).to.contain(Form);
	});
});
