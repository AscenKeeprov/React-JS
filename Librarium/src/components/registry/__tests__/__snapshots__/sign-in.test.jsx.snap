// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign in form renders without crashing 1`] = `ShallowWrapper {}`;

exports[`Sign in view renders without crashing 1`] = `ShallowWrapper {}`;
